controllers/model/alertmanager_resources.go

```diff
@@ -29,17 +36,9 @@ func GetAlertmanagerTLSSecret(cr *v1.Observability) *v13.Secret {
 }
 
 func GetAlertmanagerRoute(cr *v1.Observability) *routev1.Route {
-	if cr.Spec.SelfContained != nil && cr.Spec.SelfContained.AlertManagerRoute != "" {
-		return &routev1.Route{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      cr.Spec.SelfContained.AlertManagerRoute,
-				Namespace: cr.Namespace,
-			},
-		}
-	}
```

Comment:
While I understand removing needing `AlertManagerRoute` if we set to default name in general, I wonder is there any merit in keeping this anyway? :thinking: 

For example, we will need to set the `AlertManagerDefaultName` to be `alertmanager-route` if we would like to keep the same route name in AMO for OO but having `alertmanager-route` as a resource name that is not the route may be confusing.

The other side is that is if set the `AlertManagerDefaultName` to be different to AMO, then this would affect the route name and test case / SOPs would need to take this account and require them to be updated also :thinking:

Reply:
I get what your saying and I can concatenate  `-route` to the string at creation but that will effect OO installation in kafka as well, which I am not going to do. The idea here is to reduce the amount of overrides we need to set and not setting a different string for each object created.  
There is already jira's for checking the SOP and test cases